applications/luci-app-cloudflared/root/usr/share/rpcd/acl.d/luci-app-cloudflared.json

```diff
@@ -0,0 +1,16 @@
+{
+	"luci-app-cloudflared": {
+		"description": "Grant access to Cloudflared configuration",
+		"read": {
+			"uci": [ "cloudflared" ],
+			"ubus": {
+				"service": [ "list" ]
+			},
+			"file": [ "/var/log/cloudflared.log" ]
+		},
+		"write": {
+			"uci": [ "cloudflared" ],
+			"file": [ "/etc/cloudflared/" ]
```

Comment:
```js
			"file": [ "/etc/cloudflared/*" ]
```

Although I'm hesitant to do this since something like `"/etc/cloudflared/../../"` could be possible. Which is why writes are (should be) generally granted on a file level.

Test locally and see whether you can escalate using the existing `"/etc/cloudflared/"` permission and a write from within js to some other folder like `"/etc/cloudflared/../../"`

Reply:
Checked: a user with ACL for the CF can't change any file or upload a new outside of the /etc/cloudflared/.